### PR TITLE
Add content on why port status is helpful

### LINKED ISSUE #89: Port Status pages do not describe why port status is useful
There is lots of detailed descriptions of _what_ the `PortStatus` operator does on this template:

https://github.com/open-ephys/bonsai-onix1-docs/blob/2e74b86d3b16218b2d8ba141cf284ccb5ccc9369/template/partials/hardware/portStatus.tmpl.partial#L2

There is no real-world information on why this is useful .e.g. if you will know  if communication fails and for what reasons in order to get help. Or, exactly when the animal bit through the tether

### PR DESCRIPTION
**Pull request by cjsha**
Resolves #89

### CODE REVIEW

File: template/partials/hardware/portStatus.tmpl.partial [hunk 1]
```diff
@@ -1,13 +1,13 @@
 <p>
-  The following excerpt from the {{{hardware}}} <a class="xref" href="~/articles/hardware/{{{oe.hardwareDirectory}}}/overview.html">example workflow</a> demonstrates port status functionality and saves timestamped port status data.
+  The {{{hardware}}} reports when a port connection enters or leaves an aberrant state. This is helpful for diagnostic purposes - knowing the time and and type of a communication failure can facilitate tracking down its cause. The following excerpt from the {{{hardware}}} <a class="xref" href="~/articles/hardware/{{{oe.hardwareDirectory}}}/overview.html">example workflow</a> demonstrates port status functionality and saves timestamped port status data.
```
Comment (jonnew) [inline template/partials/hardware/portStatus.tmpl.partial:2]: Altered text:

The {{{hardware}}} reports when a headstage port connection enters or leaves an aberrant state. This is helpful for diagnostic purposes.  Knowing the time and and type of a headstage communication failure is a good first step to facilitate tracking down its cause.

File: template/partials/hardware/portStatus.tmpl.partial [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+  Port status data is produced in Bonsai by the <a class="xref" href="~/api/OpenEphys.Onix1.PortStatus.html">PortStatus</a> operator which generates a sequence of <a class="xref" href="~/api/OpenEphys.Onix1.PortStatus.html">PortStatusFrames</a>. <code>PortStatus</code> emits a <code>PortStatusFrame</code> when the SERDES loses or gains lock, the SERDES fails or passes parity check, or the <a class="xref" href="~/api/OpenEphys.Onix1.PortStatusCode.html">PortStatusCode</a> changes value. The <code>DeviceName</code> is set to "{{{hardwareOperator}}}/PortController". This links the <code>PortStatus</code> operator to the corresponding configuration operator.
```
Comment (jonnew) [inline template/partials/hardware/portStatus.tmpl.partial:10]: Headstage port status data is produced in Bonsai by the <a class="xref" href="~/api/OpenEphys.Onix1.PortStatus.html">PortStatus</a> operator which generates a sequence of <a class="xref" href="~/api/OpenEphys.Onix1.PortStatus.html">PortStatusFrames</a>. <code>PortStatus</code> emits a <code>PortStatusFrame</code> when the headstage loses or gains communication lock, and chip level partiy error is detected, a packet is badly formatted, or a CRC error is encountered indicating that data has been corrupted during transmission, etc. The <a class="xref" href="~/api/OpenEphys.Onix1.PortStatusCode.html">PortStatusCode</a> indicates which issue has been encountered. 

relevant: https://github.com/open-ephys/bonsai-onix1/issues/351 to above explaination

This should be at the top it has nothing to do with this paragraph:

The <code>DeviceName</code> is set to "{{{hardwareOperator}}}/PortController". This links the <code>PortStatus</code> operator to the corresponding configuration operator.

Comment (cjsha) [reply template/partials/hardware/portStatus.tmpl.partial]: I think I adequately address this, but I advise a double-check to confirm. Please resolve conversation if so.

File: template/partials/hardware/portStatus.tmpl.partial [hunk 3]
```diff
@@ -1,20 +1,37 @@
...
+  <code>PortStatusFrame</code> whenever <a class="xref"
+  href="~/api/OpenEphys.Onix1.PortStatusCode.html">PortStatusCode</a> changes value. <code>PortStatus</code>'s
+  <code>DeviceName</code> property is set to "{{{hardwareOperator}}}/PortController". This links the
+  <code>PortStatus</code> operator to the breakout board's port controller.
 </p>
 
 <p>
-  The <a class="xref" href="https://bonsai-rx.org/docs/api/Bonsai.Reactive.Timestamp.html">TimeStamp</a> operator generates a sequence of timestamped items from its input sequence. The <a class="xref" href="https://bonsai-rx.org/docs/api/Bonsai.IO.CsvWriter.html">CsvWriter</a> operator writes <code>Timestamp</code> as well as <code>Clock</code>, <code>StatusCode</code>, <code>SerdesLocked</code>, and <code>SerdesPass</code> members from <code>PortStatusFrame</code> to a file with the following name format: <code>port-status_&lt;timestamp&gt;.csv</code>.
+  The <a class="xref" href="https://bonsai-rx.org/docs/api/Bonsai.Reactive.Timestamp.html">TimeStamp</a> operator
+  generates a sequence of timestamped items from its input sequence. The <a class="xref"
```
Comment (jonnew) [inline template/partials/hardware/portStatus.tmpl.partial:25]: timestamped -> UTC timestamped

File: template/partials/hardware/portStatus.tmpl.partial [hunk 4]
```diff
@@ -1,20 +1,37 @@
 <p>
-  The following excerpt from the {{{hardware}}} <a class="xref" href="~/articles/hardware/{{{oe.hardwareDirectory}}}/overview.html">example workflow</a> demonstrates port status functionality and saves timestamped port status data.
+  The Onix system reports when a port connection enters or leaves an aberrant state. Such aberrant states include loss
```
Comment (jonnew) [inline template/partials/hardware/portStatus.tmpl.partial:2]: port -> headstage port

File: template/partials/hardware/portStatus.tmpl.partial [hunk 5]
```diff
@@ -1,20 +1,37 @@
...
+  <code>DeviceName</code> property is set to "{{{hardwareOperator}}}/PortController". This links the
+  <code>PortStatus</code> operator to the breakout board's port controller.
 </p>
 
 <p>
-  The <a class="xref" href="https://bonsai-rx.org/docs/api/Bonsai.Reactive.Timestamp.html">TimeStamp</a> operator generates a sequence of timestamped items from its input sequence. The <a class="xref" href="https://bonsai-rx.org/docs/api/Bonsai.IO.CsvWriter.html">CsvWriter</a> operator writes <code>Timestamp</code> as well as <code>Clock</code>, <code>StatusCode</code>, <code>SerdesLocked</code>, and <code>SerdesPass</code> members from <code>PortStatusFrame</code> to a file with the following name format: <code>port-status_&lt;timestamp&gt;.csv</code>.
+  The <a class="xref" href="https://bonsai-rx.org/docs/api/Bonsai.Reactive.Timestamp.html">TimeStamp</a> operator
+  generates a sequence of timestamped items from its input sequence. The <a class="xref"
+  href="https://bonsai-rx.org/docs/api/Bonsai.IO.CsvWriter.html">CsvWriter</a> operator writes <code>Timestamp</code> as
+  well as <code>Clock</code>, <code>StatusCode</code>, <code>SerdesLocked</code>, and <code>SerdesPass</code> members
```
Comment (jonnew) [inline template/partials/hardware/portStatus.tmpl.partial:27]: I'm going to wrap these all into the `StatusCode`. I would just remove mention of the locked and pass elements to save yourself a headache. 

https://github.com/open-ephys/bonsai-onix1/issues/351

**Review by jonnew — APPROVED** [review]
Left a few comments, looks great.

File: template/partials/hardware/portStatus.tmpl.partial (hunk 5 above)
Comment (cjsha) [reply template/partials/hardware/portStatus.tmpl.partial:27]: thanks for catching that

**Pull request merged**